fuzzers/005-tilegrid/iob/top.py

```diff
@@ -11,24 +11,19 @@
 
 
 def gen_iobs():
-    '''
-    IOB33S: main IOB of a diff pair
-    IOB33M: secondary IOB of a diff pair
```

Comment:
`IOB33S` was actually the main? I would have guess the `S` and `M` stood for `Master`, `Slave` or `Main`, `Secondary`.